woocommerce/woocommerce-android
# Unified Login: Not a WordPress Site error flow

Closes #2942. This PR adds the new "Not a WordPress site" error screen to the login by store address flow. This screen will only be show when the user enters a URL for a site that does not have WordPress installed. 

Before | After
-- | --
![not-store-before](https://user-images.githubusercontent.com/5810477/102275709-d36c5780-3ef3-11eb-98f6-2f289b86fe11.png)|![screenshot-1608068020199](https://user-images.githubusercontent.com/5810477/102275717-d49d8480-3ef3-11eb-89d3-d73c1ba10e64.jpg)

### Tracks events

No new tracks events added, only new `Step` and `Flow` properties. Below are the events associated with this new view:

- Not a WordPress site error screen displayed. Set the active step to `NOT_WORDPRESS_SITE`:
`Tracked: unified_login_step, Properties: {"source":"default","flow":"login_site_address","step":"not_wordpress_site"}`

- On the new screen: User clicked on the "Help" icon:
`Tracked: unified_login_interaction, Properties: {"source":"default","flow":"login_site_address","step":"not_wordpress_site","click":"show_help"}`

- On the new screen: User clicked "Try another store":
`Tracked: unified_login_interaction, Properties: {"source":"default","flow":"login_site_address","step":"not_wordpress_site","click":"try_another_store"}`

- On the new screen: User clicked "Try another account":
`Tracked: unified_login_interaction, Properties: {"source":"default","flow":"login_site_address","step":"not_wordpress_site","click":"try_another_account"}`


### Testing
Do a fresh install of the app or clear app data, and start the login process:
1. Click **Enter your store address**. The site address form is displayed. 
2. Enter `google.com` in the store address field and click **Continue**. The "Not a WordPress site" error screen is displayed. 
3. Click the **Help** icon and verify the help screen is displayed. Verify the appropriate tracks event is fired off. 
4. Click **back** to return to the error screen.
5. Click **Try another store**. Verify the site address form is displayed and the appropriate tracks event is fired off.
6. Click **Continue** to re-display the "Not a WordPress site" error screen.
7. Click **Log in with another account**. Verify the app is directed back to the Prologue/Welcome screen and the appropriate tracks event is fired off.

Update release notes:
